samples/performance/subpasses/subpasses_tutorial.md

```diff
@@ -54,59 +54,58 @@ vec4 world_w = inv_view_proj * clip;
+
+It is important to note that by using a `VkImageFormat` that requires more bits, it is most likely that the G-buffer will no longer fit the drivers budget, denying the possibility to merge the subpasses. The following picture shows how the number of physical tiles used almost doubles to `409.6k/s`, indicating that subpasses are not merged.
 
 ![G-buffer size](images/gbuffer-size.jpg)
 
 ## Transient attachments
 
-Some framebuffer attachments, like _depth_, _albedo_, and _normal_ in the sample, are cleared at the beginning of the render pass, written by the geometry subpass, read by the lighting subpass, and discarded at the end of the render pass. If the GPU has enough memory available to store them on tile memory, there is no need to write them back to external memory. Actually, there is not even need to allocate them at all.
-
-In practice, their [image usage](https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/VkImageUsageFlagBits.html) needs to be specified as `TRANSIENT` and their [memory](https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/VkMemoryPropertyFlagBits.html) needs to be `LAZILY_ALLOCATED`. Failing to set these flags properly will lead to an increase of [fragment jobs](https://community.arm.com/developer/tools-software/graphics/b/blog/posts/mali-bifrost-family-performance-counters) as the GPU will need to write them back to external memory.
+Some framebuffer attachments in the sample (such as _depth_, _albedo_, and _normal_), are cleared at the beginning of the render pass, written by the geometry subpass, read by the lighting subpass, and discarded at the end of the render pass. If the GPU has enough memory available to store them on tile memory, there is no need to write them back to external memory. Actually, there is not even a need to allocate them at all.
 
 ![Non-transient attachments](images/transient-attachments.jpg)
 
+In practice, their [image usage](https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/VkImageUsageFlagBits.html) needs to be specified as `TRANSIENT` and their [memory](https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/VkMemoryPropertyFlagBits.html) needs to be `LAZILY_ALLOCATED`. Failing to set these flags properly will lead to an increase of [fragment jobs](https://community.arm.com/developer/tools-software/graphics/b/blog/posts/mali-bifrost-family-performance-counters) as the GPU will need to write them back to external memory. As you can see in the above screenshot, we see roughly a double in fragment jobs per second (from `56/s` to `113/s`).
```

Comment:
Is this correct? I would expect transient buffers to affect bandwidth, but not the number of fragment jobs.

Reply:
AFAIK when it comes to transient vs non-transient, as long as the image is storeOp=DONT_CARE, there's rarely any change to the BW. Transient only directly impacts the memory footprint which has nothing to do with bandwidth consumption.

Reply:
Yes, I should have written memory, not bandwidth. My bad.
I still can't see how it affects the number of fragment jobs though.

Reply:
I think the confusion might be in the definition of fragment jobs

Reply:
In Mali we sometimes schedule fragment jobs to do some setup/metadata-management, but these are often tiny single-thread type jobs poking some value into memory.

Reply:
What is the Broadcom equivalent of this? Are you seeing the fragment jobs not increasing at all with the respective setup?

Reply:
For us, fragment jobs represent the number of rendering tasks that the GPU has been given (roughly equivalent to one per  subpass in this example). Neither transient buffers or changes to the storeOp can affect this for us.

I could understand if changing from renderpasses to subpasses changed this, but the fact that it's transient buffers confuses me.

This sentence in particular: "Failing to set these flags properly will lead to an increase of [fragment jobs] ... as the GPU will need to write them back to external memory".

Reply:
Do you see no change in the fragment job counter?

Reply:
Can I suggest, seeing as this is just to rename the sample, that we move this to an issue to be resolved at a later stage. This paragraph exists in master.

Reply:
No change in the counter for us, no.

> move this to an issue to be resolved at a later stage

Sure. No problem.